janitor/functions.py

```diff
@@ -64,27 +64,27 @@
 )
 
 
-def unionize_dataframe_categories(
-    *dataframes, column_names: Optional[Iterable[pd.CategoricalDtype]] = None
+def unionize_DataFrame_categories(
```

Comment:
DataFrame doesn't need to be changed in function names or inputs. Please revert this to `unionize_dataframe_cartegories`. We want to follow the Variable and Function naming from [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) here.